src/com/lftechnology/java/training/alina/jdbc/controller/EmployeeController.java

```diff
@@ -0,0 +1,236 @@
+     * @author Alina Shakya <alinashakya@lftechnology.com>
+     */
+    public static void deleteExistingEmployee(Scanner scanner) {
+        Employee employee = new Employee();
+        employee.setFullname(UtilityService.getInputData(scanner, "Enter fullname to delete : "));
+        boolean isDeleted = employeeDao.delete(employee.getFullname());
+        if (isDeleted) {
+            LOGGER.log(Level.INFO, "\n=====>\nSuccessfully deleted employee with Name {0}.\n=====>\n",
+                    new Object[] { employee.getFullname() });
+        } else {
+            LOGGER.log(Level.WARNING, "\n=====>\nFailed to delete an employee with Name {0}.\n=====>\n",
+                    new Object[] { employee.getFullname() });
+        }
+    }
+
+    /**
+     * Terminate existing employee
+     * 
+     * @param scanner
+     *            {@link Scanner}
+     * @author Alina Shakya <alinashakya@lftechnology.com>
+     */
+    public static void terminateExistingEmployee(Scanner scanner) {
+        Employee employee = new Employee();
+        employee.setFullname(UtilityService.getInputData(scanner, "Enter fullname : "));
+        boolean isDeleted = userDao.delete(employee.getFullname());
+        if (isDeleted) {
+            LOGGER.log(Level.INFO, "\n=====>\nSuccessfully terminated employee with Name {0}.\n=====>\n",
+                    new Object[] { employee.getFullname() });
+        } else {
+            LOGGER.log(Level.WARNING, "\n=====>\nFailed to terminate an employee with Name {0}.\n=====>\n",
+                    new Object[] { employee.getFullname() });
+        }
+    }
+
+    /**
+     * Gets Employee lists soreted by alphabetical order
+     * 
+     * @author Alina Shakya <alinashakya@lftechnology.com>
+     */
+    public static void getEmployeeList() {
+        List<Employee> list = employeeDao.findAll();
+        Collections.sort(list);
+        if (list.size() > 0) {
+            LOGGER.log(
+                    Level.INFO,
+                    "\n<=====>\nNumber of Employee : {0} \n<=====>\n\n-----------------------------------------------------------------------------------------------\n{1}",
+                    new Object[] { list.size(), list });
+        } else {
+            LOGGER.log(Level.INFO, "\n<=====>\nNumber of Employee : {0}\n----------------------\nNo Record Found.\n<=====>\n\n",
+                    new Object[] { list.size() });
+        }
+    }
+
+    /**
+     * Search existing employee from the list
+     * 
+     * @param scanner
+     *            {@link Scanner}
+     * @author Alina Shakya <alinashakya@lftechnology.com>
+     */
+    public static void searchExistingEmployee(Scanner scanner) {
+        String searchContent = UtilityService.getInputData(scanner, "Search Employee by fullname, department or address : ");
+        List<Employee> list = employeeDao.searchEmployee(searchContent, searchContent, searchContent);
+        Collections.sort(list);
+        if (list.size() > 0) {
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Use isEmpty() to check whether the collection is empty or not. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://nemo.sonarqube.org/coding_rules#rule_key=squid%3AS1155)